### PR TITLE
[REVIEW] Update Azure MNMG notebook to use Azure Opendataset

### PR DESCRIPTION
**Pull request by Nanthini10**
- Update data source to azure open-dataset
- Update `security=True`

### CODE REVIEW

File: azure/notebooks/azure_mnmg.ipynb [hunk 1]
```diff
@@ -69,16 +71,15 @@
...
     "location = \"SOUTH CENTRAL US\"\n",
-    "resource_group = \"RAPIDS-HPO-test\"\n",
+    "resource_group = \"RAPIDS-mnmg\"\n",
     "vnet = \"dask-vnet\"\n",
-    "security_group = \"test-security-group\"\n",
+    "security_group = \"dask-nsg\"\n",
     "\n",
     "vm_size = \"Standard_NC12s_v3\"\n",
-    "docker_image = \"rapidsai/rapidsai-core:cuda10.2-runtime-ubuntu18.04-py3.8\"\n",
+    "docker_image = \"rapidsai/rapidsai-nightly:0.18-cuda10.2-runtime-ubuntu18.04-py3.8\"\n",
```
Comment (zronaghi) [inline azure/notebooks/azure_mnmg.ipynb:79]:
```suggestion
    "docker_image = \"rapidsai/rapidsai-nightly:0.18-cuda11.0-runtime-ubuntu18.04-py3.8\"\n",
```

Comment (zronaghi) [reply azure/notebooks/azure_mnmg.ipynb:79]: Do we need to modify the README to use the same container?

File: azure/notebooks/azure_mnmg.ipynb [hunk 2]
```diff
@@ -334,44 +304,47 @@
    "outputs": [],
    "source": [
     "with Client(cluster) as client:\n",
-    "    \n",
     "    print(\"Start Workflow\")\n",
-    "    cluster.scale(2)\n",
-    "    client.wait_for_workers(4)\n",
+    "    cluster.scale(n_workers)\n",
+    "    client.wait_for_workers(n_workers * 2)\n",
```
Comment (zronaghi) [inline azure/notebooks/azure_mnmg.ipynb:309]: Great, maybe add a comment that 2 is number of GPUs per node?

**Review by zronaghi — COMMENTED** [review]
Looks great, couple of minor comments

File: azure/notebooks/azure_mnmg.ipynb (hunk 1 above)
Comment (Nanthini10) [reply azure/notebooks/azure_mnmg.ipynb:79]: Yes, thank you. I missed that.

**Comment by zronaghi** [comment]
Thanks, closes #62

**Review by zronaghi — APPROVED** [review]

**Comment by mreyesgomez** [comment]
Great thank you

**Pull request merged**